XenCert/StorageHandler.py

```diff
@@ -2781,103 +2800,274 @@ def MetaDataTests(self):
         Print("MetaDataTests not applicable to NFS SR type.")
         return False
 
+    def PoolTests(self):
```

Comment:
Or we could change the definition of pool tests for NFS SRs. Currently "This test creates shared SRs and verifies that PBD records display the same number of paths for each host in the pool." This is accomplished by enabling multipathing and counting the number of paths on each host. For NFS SRs, we could just check for the availability of the shared SR.